### ISSUE TITLE
Skipping a media segment that goes to the end of the file doesn't mark the file played.

### ISSUE THREAD

**Issue by reacocard** [jellyfin/jellyfin-androidtv#4165]
### Describe the bug

1. Have an episode in Jellyfin that has a media segment covering from a point after "Minimum Resume Percentage" but before "Maximum Resume Percentage", to the end of the file.
2. Turn on ask to skip for that segment's type.
3. Play the file, and on being prompted to skip the segment, skip it.
4. Observe that Jellyfin offers to resume from the point of the skip, rather than marking the file played.

I hit this on a series with very short episodes, so the min/max resume was way shorter than the ending credits.

### Logs

This bit of logcat looks relevant - it's submitting the time of the skip rather than the time it tried to seek to.

```
11-04 22:23:12.646 20015 20015 D PlaybackController: Trying to seek from 64316 to 137191
11-04 22:23:12.646 20015 20015 D PlaybackController: Container: ts
11-04 22:23:12.647 20015 20015 D PlaybackController: stop called at 64377
11-04 22:23:12.650 20015 20015 I ReportingHelper$reportStopped: Reporting EPISODE_TITLE playback stopped at 643770000
```

### Application version

compiled from commit f474e2a

### Where did you install the app from?

Sideloaded APK

### Device information

Nvidia Shield Pro 2017

### Android version

Android 11, 2022-04-05 patch

### Jellyfin server version

10.10.0

**Comment by nielsvanvelzen** [comment]
I think I messed up something when I cleaned up my debug logging in the initial fix. Should now be fixed for real with the next beta 😄

**Comment by nielsvanvelzen** [comment]
Ok another attempt, this should finally fix it: #4197. I'll wait with merging until you've confirmed it to work this time 😅
